### ISSUE TITLE
`ninja par_runner`: target does not exist

### ISSUE THREAD

**Issue by puradawid** [piotr022/UV_K5_playground#14]
Is this ninja's target still on the table:

```
$ ninja par_runner
ninja: error: unknown target 'par_runner'
```

and when I do a build without any target:

```
$ ninja
[22/26] Linking CXX executable src\most_useless_mod\most_useless_mod
Memory region         Used Size  Region Size  %age Used
             RAM:          65 B        256 B     25.39%
           FLASH:       61232 B        60 KB     99.66%
   text    data     bss     dec     hex filename
   2372   58864      61   61297    ef71 most_useless_mod
generating full binary with bootloader to most_useless_mod_with_bootloader.bin
[23/26] Linking CXX executable src\pong\pong
Memory region         Used Size  Region Size  %age Used
             RAM:         105 B        256 B     41.02%
           FLASH:       61392 B        60 KB     99.92%
   text    data     bss     dec     hex filename
   2532   58864     101   61497    f039 pong
generating full binary with bootloader to pong_with_bootloader.bin
[24/26] Linking CXX executable src\rssi_printer\rssi_printer
Memory region         Used Size  Region Size  %age Used
             RAM:         145 B        256 B     56.64%
           FLASH:       61288 B        60 KB     99.75%
   text    data     bss     dec     hex filename
   2428   58868     134   61430    eff6 rssi_printer
generating full binary with bootloader to rssi_printer_with_bootloader.bin
[25/26] Linking CXX executable src\rssi_sbar\rssi_sbar
Memory region         Used Size  Region Size  %age Used
             RAM:          45 B        256 B     17.58%
           FLASH:       61292 B        60 KB     99.76%
   text    data     bss     dec     hex filename
   2432   58868      37   61337    ef99 rssi_sbar
generating full binary with bootloader to rssi_sbar_with_bootloader.bin
[26/26] Linking CXX executable src\spectrum\spectrum
Memory region         Used Size  Region Size  %age Used
             RAM:          92 B        256 B     35.94%
           FLASH:       61408 B        60 KB     99.95%
   text    data     bss     dec     hex filename
   2548   58864      88   61500    f03c spectrum
```

`cmake` works well. I have been trying to check your last commits but didn't find any changes to this target.

Thanks in advance, Dawid SP4KOT

**Comment by piotr022** [comment]
hi this looks fine, every taget has been builded. I need to update instructions, i moved par_runner to libs/k5_uv_system, and now every mod is using it, so there no need to build it as separate target because this dependency is causing that par_runner is builded automaticly as static library.

Also check building via github CI, after forking my repo, set in git settings to allow running workflows. After that every time you push something to your fork main branch, and after about 2 min in repo 'actions' tab you will see builded binaries as workflow artifacts.

**Comment by piotr022** [comment]
also be carefull to not upload these binaries directly by quancheng upload tool, they are not encrypted. If you want to get ncrypted binary build specific target with _encoded at the end, for example: ninja spectrum_encoded

**Comment by puradawid** [comment]
Thanks for the reply. I have noticed I have not cloned submodules - updated only the main repo :)

For anybody that will face this issue, it's simply to do: `git submodule update --init --recursive`.

Anyway, the build looks fine now:

```
ninja pong_encoded
[1/1] cmd.exe /C "cd /D C:\Users\purad\projects\UV_K5_play...projects/UV_K5_playground/build/src/pong/pong_encoded.bin"
Saved encoded firmware to C:/Users/purad/projects/UV_K5_playground/build/src/pong/pong_encoded.bin
```

Also, I had to install `serial` dependency coming from `fw_tools` by `pip install serial`.

I am about to help SP4UBW to update the RSSI mod for "modified" radio (I think it's about removing amp transistor), so I am interested in rebuilding this one and installing it on my dev kit radios (that are not yet here anyway). 

Many thanks for your help!
